tests/trainers/README.md

```diff
@@ -0,0 +1,100 @@
+BCI tests modules
+=================
+
+Description
```

Comment:
I don't think that this fits into the README. However, this is a pretty good guide and it would be a shame not to use it. Can I include this part in the documentation of [pytest_container](https://github.com/dcermak/pytest_container/)?

Reply:
Sure, thank you!

Reply:
I moved the copy syntax description in a DOCUMENT separated from the README file containing the only test specifications. I used the md markdown formatting.